components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -649,7 +651,7 @@
                             } else {
                             %>
                             <input type="text" name=<%=value%> id=<%=value%> style="width:95%"
```

Comment:
the variable value is iterated in a loop and is defined for each iteration as 
'value = "propertyValue_" + i;'.

Both the string and the integer is defined in the JSP context, and no user input or stored parameter is injected. Hence it is unnecessary.